ChangeLog

```diff
@@ -398,6 +398,14 @@ Release date: 2020-04-27
 
 * A new check `inconsistent-quotes` was added.
 
+* New check: ``consider-using-generator``
```

Comment:
This is in `What's New in Pylint 2.5.0?`, should be in 2.7.0 :)

Reply:
The 2.7.rst is a new file, should I rebase?

Reply:
Well, yes you should rebase in order to get 2.7.rst, but also for this particular comment it should be copy paster be higher in the Changelog file :)